recipes/ocean-contracts/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "ocean-contracts" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ocean-contracts-{{ version }}.tar.gz
+  sha256: 593f34f121c987b05395fde4338c6973e6b6c9085f19c7c8747fe9bf99a4e7dc 
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - artifacts
+    - addresses 
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/oceanprotocol/contracts
+  summary: 🐳 Ocean Protocol L1 - v4
+  description: 🦑 Smart contracts for Ocean Protocol v4. https://oceanprotocol.com Ocean v4 is part of the Ocean Protocol toolset.
+  license: 'Apache-2.0'
+  license_file: license.txt
+
+extra:
+  recipe-maintainers:
+    - oceanprotocol
```

This really should include your user since it's a list of people who are responsible for maintaining the recipe.
```suggestion
    - calina-c
```